src/controllers/use-router-actions/index.ts

```diff
@@ -0,0 +1,11 @@
+import { useRouterStoreActions } from '../router-store';
+import { RouterActionsType } from '../router-store/types';
+
+/**
+ * Hook for accessing the public router store without re-rendering on route change
+ */
+export const useRouterActions = (): RouterActionsType => {
```

Comment:
FYI RSS now supports this pattern out of the box. We could kill this and change `useRouterStoreActions` to be:
const useRouterActions = createActionsHook(Store)

Reply:
Thanks for the info, I'll do that in another pull request for all of them